Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -343,8 +343,10 @@ void NetworkProcess::initializeNetworkProcess(NetworkProcessCreationParameters&&
     for (auto& scheme : parameters.urlSchemesRegisteredAsLocal)
         registerURLSchemeAsLocal(scheme);
 
+#if !PLATFORM(COCOA)
```

Comment:
This should use a more clear macro, probably !ENABLE(...), to document the reason for this exclusion.  

Generally using raw PLATFORM(...) macros outside of defining other more specific macros is an anti-pattern to be avoided.